src/tflite_cpu.cpp

```diff
@@ -303,29 +302,21 @@ class MLModelServiceTFLite : public vsdk::MLModelService,
         model_path_contents_stream << in.rdbuf();
         state->model_data = std::move(model_path_contents_stream.str());
 
-        // Create an error reporter so that we can extract detailed
-        // error information from TFLite when things go wrong. The
-        // error state is protected by the interpreter lock.
-        state->model.reset(TfLiteModelCreateWithErrorReporter(
```

Comment:
The `state` struct is now an `ErrorReporter` instance, so this logic moved to the new `Report` virtual function override in `struct state`.